tst/Boolean/Assert-False.Tests.ps1

```diff
@@ -23,7 +23,7 @@ Describe "Assert-False" {
     }
 
     Context "Validate messages" {
-        It "Given value '<expected>' that is not `$false it returns expected message '<message>'" -TestCases @(
```

Comment:
you are correct here, the template should display Actual and not expected. But please use <actual> with lowecase letter.